Week3/homework/step3-bonus.js

```diff
@@ -4,7 +4,8 @@ const values = ['a', 'b', 'c', 'd', 'a', 'e', 'f', 'c'];
 
 function makeUnique(arr) {
   // Replace this comment and the next line with your code
-  console.log(arr);
+  const uniqueArray = arr.filter((item, pos) => arr.indexOf(item) === pos);
+  return uniqueArray;
```

Comment:
Very well. This logic works fine but it seems a bit too much. Try and read about a new concept in ES6 which is called Sets and see if you can use it to make this code shorter. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set